luaa.c

```diff
@@ -411,6 +417,94 @@ get_modifier_name(int map_index)
     return 0; /* \0 */
 }
 
+// Helper function for luaA_get_key_name() below.
```

Comment:
Please use `/*` for the comments for consistency. There is some `//`, but 99% of the codebase (and most C code) use `/*`.